lib/pg_sequencer/schema_dumper.rb

```diff
@@ -1,16 +1,16 @@
 module PgSequencer
   module SchemaDumper
     extend ActiveSupport::Concern
-    
+
     included do
       alias_method_chain :tables, :sequences
     end
-    
+
     def tables_with_sequences(stream)
       tables_without_sequences(stream)
       sequences(stream)
     end
-    
+
     private
     def sequences(stream)
```

Comment:
if the stream isn't being passed around any longer, should we remove this argument?